tests/basictests/dsp-operator.sh

```diff
@@ -120,16 +143,54 @@ EOF
     os::cmd::try_until_not_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/runs/${RUN_ID} | jq '" "null" $odhdefaulttimeout $odhdefaultinterval
 }
 
+function create_experiment_run() {
+    header "Creating a run that uses the test experiment"
+
+    RUN_ID_EXPT=$((curl -k -H "Authorization: Bearer ${SA_TOKEN}" \
+        -H "Content-Type: application/json" \
+        -X POST "https://${ROUTE}/apis/v1beta1/runs" \
+        -d @- << EOF
+        {
+            "name": "test-experiment-run",
+            "description": "This is a test run that uses the test experiment",
+            "pipeline_spec":{
+                "pipeline_id":"${PIPELINE_ID}"
+            },
+            "resource_references":[
+            {
+               "key":{
+                  "type":"EXPERIMENT",
+                  "id":"${EXPERIMENT_ID}"
+               },
+               "name":"Default",
+               "relationship":"OWNER"
+            }
+            ]
+        }
+EOF
+        ) | jq -r .run.id)
+
+    os::cmd::try_until_not_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/runs/${RUN_ID_EXPT} | jq '" "null" $odhdefaulttimeout $odhdefaultinterval
+}
+
 function verify_run_availabilty() {
     header "verify the run exists"
 
-    os::cmd::try_until_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/runs | jq '.total_size'" "1" $odhdefaulttimeout $odhdefaultinterval
+    os::cmd::try_until_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/runs | jq '.total_size'" "2" $odhdefaulttimeout $odhdefaultinterval
 }
 
 function check_run_status() {
     header "Checking run status"
 
     os::cmd::try_until_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/runs/${RUN_ID} | jq '.run.status'" "Completed" $odhdefaulttimeout $odhdefaultinterval
+    os::cmd::try_until_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/runs/${RUN_ID_EXPT} | jq '.run.status'" "Completed" $odhdefaulttimeout $odhdefaultinterval
```

Comment:
In the `delete_runs()` section
we should delete this run as well RUN_ID_EXPT.

Reply:
Done, added commands to delete the RUN_ID_EXPT run.